src/Stack/Types/Config.hs

```diff
@@ -356,7 +356,7 @@ data Config =
          -- match cabal.
          ,configDefaultTemplate     :: !(Maybe TemplateName)
          -- ^ The default template to use when none is specified.
-         -- (If Nothing, the default default is used.)
+         -- (If Nothing, the default is used.)
```

Comment:
Sometimes doubling is intentional, but, this appears to be a comment w/ normal English...